packages/v/vanta.json

```
@@ -0,0 +1,33 @@
+{
+  "name": "vanta",
+  "description": "3D animated backgrounds for your website",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/tengbao/vanta"
+  },
+  "keywords": [
+    "animation",
+    "3d",
+    "three",
+    "threejs",
+    "background",
+    "canvas",
+    "javascript",
+    "webgl"
+  ],
+  "author": "Teng Bao",
+  "license": "MIT",
+  "npmName": "vanta",
+  "autoupdate": {
+    "source": "npm",
+    "target": "vanta",
+    "fileMap": [
+      {
+        "basePath": "dist",
+        "files": [
+          "**/*"
```

```suggestion
          "*.js"
```
We only need the `js` files from `dist`, not anything else that might get added later on.